addons/common/functions/fnc_getMapData.sqf

```diff
@@ -129,4 +129,4 @@ if (_map in ["xcam_prototype"]) exitWith { [35, 0] };
 if (_map in ["xcam_taunus"]) exitWith { [50, 0] };
 if (_map in ["zargabad"]) exitWith { [35, 2000] };
 
-[] //Return empty array if we have no specific data for the map
+[getNumber (configFile >> "CfgWorlds" >> _map >> "latitude"), 0] // Use terrain config if we have no data for this map
```

Comment:
I think `[]` is fine to return here

https://github.com/acemod/ACE3/blob/master/addons/common/functions/fnc_getMGRSdata.sqf#L22-L33
is the only thing that uses it and it handles empty array already
+besides a debug function to test which maps are missing entries in the look up

we at least want negative lat and `elevationOffset` here to be constant